src/CodeStyle/XliffFiles.php

```diff
@@ -75,11 +75,34 @@ private static function checkUnit(Crawler $unit, array &$fixed)
                 $fixed[] = 'id of "'.substr(strtr($sourceTxt, array("\n" => "\\n")), 0, 128).'"';
             }
         }
-        if (false !== strpos($sourceTxt, ' %') && false === strpos($unit->filter('target')->text(), ' %')) {
+        if (false !== strpos($sourceTxt, ' %') && false === self::getTranslatedParamPos($unit->filter('target')->text())) {
             $fixed[] = 'TODO include parameters in source "'.strtr($sourceTxt, array("\n", "\\n")).'" (from target )';
         }
     }
 
+    /**
+     * Check if translation contains something looking like a parameter.
+     *
+     * The translated text can also contain the parameter inside quotes.
+     *
+     * @return int|false
+     */
+    private static function getTranslatedParamPos($string, $offset = 0)
+    {
+        $pos2 = strpos($string, '%', $offset);
+        if (false === $pos2 || 0 === $pos2) {
+            // not found, or at start
+            return $pos2;
+        }
+        $atPos1 = strtolower(substr($string, $pos2 - 1, 1));
+        if ($atPos1 < 'a' || $atPos1 > 'z') {
+            return $pos2 - 1;
```

Comment:
exactly seen, why -1?
I know, we normally return the position of ' %', which is one before %. But do we care about the number anywherer?